Doc/examples/Proux_et_al_2002_Figure_6.py

```diff
@@ -31,23 +31,55 @@
 
 A_rec = SeqIO.read("NC_002703.gbk", "gb")
 B_rec = SeqIO.read("AF323668.gbk", "gb")
-C_rec = SeqIO.read("NC_003212.gbk",
-                   "gb")[2587879:2625807].reverse_complement(name=True)
+C_rec = SeqIO.read("NC_003212.gbk", "gb")[2587879:2625807].reverse_complement(name=True)
 records = {rec.name: rec for rec in [A_rec, B_rec, C_rec]}
 
 # Here we hard code the gene colors for simiplicity and to match the target image.
 # In practice you might have an automatic mapping based on the gene annotation
 # or some other classification:
 
-A_colors = [red] * 5 + [grey] * 7 + [orange] * 2 + [grey] * 2 + [orange] \
-    + [grey] * 11 + [green] * 4 + [grey] + [green] * 2 + [grey, green] \
-    + [brown] * 5 + [blue] * 4 + [lightblue] * 5 + [grey, lightblue] \
-    + [purple] * 2 + [grey]
-B_colors = [red] * 6 + [grey] * 8 + [orange] * 2 + [grey] + [orange] \
-    + [grey] * 21 + [green] * 5 + [grey] + [brown] * 4 + [blue] * 3 \
-    + [lightblue] * 3 + [grey] * 5 + [purple] * 2
-C_colors = [grey] * 33 + [green] * 5 + [brown] * 4 + [blue] * 2 \
-    + [grey, blue] + [lightblue] * 2 + [grey] * 8
+A_colors = (
+    [red] * 5
+    + [grey] * 7
+    + [orange] * 2
+    + [grey] * 2
+    + [orange]
+    + [grey] * 11
+    + [green] * 4
+    + [grey]
+    + [green] * 2
+    + [grey, green]
+    + [brown] * 5
+    + [blue] * 4
+    + [lightblue] * 5
+    + [grey, lightblue]
+    + [purple] * 2
+    + [grey]
+)
+B_colors = (
+    [red] * 6
+    + [grey] * 8
+    + [orange] * 2
+    + [grey]
+    + [orange]
+    + [grey] * 21
+    + [green] * 5
+    + [grey]
+    + [brown] * 4
+    + [blue] * 3
+    + [lightblue] * 3
+    + [grey] * 5
+    + [purple] * 2
+)
+C_colors = (
+    [grey] * 33
+    + [green] * 5
+    + [brown] * 4
+    + [blue] * 2
+    + [grey, blue]
+    + [lightblue] * 2
+    + [grey] * 8
+)
```

Comment:
Personally, I would use the parentheses to get rid of the line continuation backshlashes, but would turn black off here. On my 17'' laptop, these three lists fill the whole screen.

Reply:
I agree on parentheses over slash line continuation, but actually don't mind black taking a screen's worth of vertical space for defining these lists - they are a key part of the graphical example.